docs/api/index.md

```diff
@@ -1526,7 +1526,9 @@ setData(data: Record<string, any>): Promise<void>
 
 `setData` does not allow setting new properties that are not defined in the component.
 
+::: warning
 Also, notice that `setData` does not modify composition API `setup()` data.
+:::
```

Comment:
Is that a special syntax that Github supports? It does not look like it does in the rich diff.

Reply:
Ha my bad, I thought it was in the readme, not in the docs. Looks good then